openssl-sys/src/lib.rs

```diff
@@ -264,35 +73,10 @@ pub struct X509V3_CTX {
     // Maybe more here
 }
 
-#[repr(C)]
-pub struct GENERAL_NAME {
-    pub type_: c_int,
-    pub d: *mut c_void,
-}
-
-impl Copy for GENERAL_NAME {}
-impl Clone for GENERAL_NAME {
-    fn clone(&self) -> GENERAL_NAME { *self }
-}
-
-impl Copy for X509V3_CTX {}
-impl Clone for X509V3_CTX {
-    fn clone(&self) -> X509V3_CTX { *self }
-}
-
-#[repr(C)]
-pub struct BIGNUM {
-    pub d: *mut c_void,
-    pub top: c_int,
-    pub dmax: c_int,
-    pub neg: c_int,
-    pub flags: c_int,
-}
-
-impl Copy for BIGNUM {}
-impl Clone for BIGNUM {
-    fn clone(&self) -> BIGNUM { *self }
-}
+#[cfg(target_pointer_width = "64")]
```

Comment:
The conditions in openssl's definition seem different/more complex. Are these equivalent? https://github.com/openssl/openssl/blob/6f0ac0e2f27d9240516edb9a23b7863e7ad02898/crypto/bn/asm/x86_64-gcc.c#L67